Cheat Engine/LuaHandler.pas

```diff
@@ -11144,34 +11144,44 @@ function lua_openFileAsProcess(L: Plua_State): integer; cdecl;
   oldprocessname: string;
   oldprocess: dword;
   oldprocesshandle: thandle;
+  paramaters: integer;

```

Comment:
paramaters 🤦‍♂️, auto-complete from notepad++ picked this word from line 2821 🤦‍♂️